### ISSUE TITLE
Update sets

### ISSUE THREAD

**Issue by vr-fox** [thtliife/servicenow-sync#10]
I have been using this plugin a lot, but I have found a problem. Is there a way to select the "update set" the plugin is uploading?

In a big project, we need to change "update sets" continuously tp work on different part, but is uploading to the default set

**Comment by thtliife** [comment]
Hi @vr-fox
Glad to hear the a plugin is helpful to you...
I know from experience that any changes made via ServiceNow-Sync will be pushed into whichever update set you currently have active on a standard ServiceNow instance. 
However,  next version of this plugin is a fairly major rewrite, with one of the features I am attempting being an update set and application picker. 
Similar to the one found in service-now. 
I wonder if that might be what you are looking for?

**Comment by vr-fox** [comment]
Yes, that's what I'm looking for, any preview date for the new plugin? Also if I can help, I'll be glad

**Comment by thtliife** [comment]
ETA for the new version is 3-4 weeks from now. 
As I am rewriting the code base, I'm not able to release features in increments unfortunately.
